docs/howto/manage-web-app-charms/integrate-web-app-charms.rst

```diff
@@ -0,0 +1,67 @@
+.. _integrate-12-factor-charms:
+
+Manage relations for a 12-factor app charm
+==========================================
+
+A charm integration can be added to your charmed 12-factor app by providing
+the integration and endpoint definition in your project file:
+
+.. code-block:: yaml
+
```

Comment:
```suggestion
    :caption: charmcraft.yaml

```

Here and for the rest. Sorry for this drive-by. We quite recently added these to the tutorials.